src/components/data/DataTable.js

```diff
@@ -306,6 +306,8 @@ const DataTable = props => {
       div({ style: { flex: 1 } }, [
         h(AutoSizer, [
           ({ width, height }) => {
+            const visibleColumns = _.filter('visible', columnSettings)
```

Comment:
FYI @nawatts. This LGTM, but tagging you as the author of https://github.com/DataBiosphere/terra-ui/pull/3152/files.